prime-router/src/test/kotlin/common/HttpClientUtilsTests.kt

```diff
@@ -0,0 +1,37 @@
+package gov.cdc.prime.router.common
+
+import io.ktor.client.HttpClient
+import junit.framework.TestCase.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import java.lang.reflect.Field
+import kotlin.test.Test
+
+class HttpClientUtilsTests {
+
+    @BeforeEach
+    fun setup() {
+        HttpClientUtils.reset()
+    }
+
+    @Test
+    fun `test default values for client is null`() {
+        // httpClient should be initialized as null
+        val httpClientField: Field = HttpClientUtils::class.java.getDeclaredField("httpClient")
+        httpClientField.trySetAccessible()
+        assertEquals(null, httpClientField.get(HttpClient::class))
+    }
+
+    @Test
+    fun `test httpClient is a singleton`() {
+        // create the httpClient singleton
+        val httpClient1 = HttpClientUtils.getDefaultHttpClient()
+        val httpClientObjRef1 = System.identityHashCode(httpClient1)
```

Comment:
I don't think this extra step to get the hash is necessary? Have you tried comparing the objects directly? iirc `==` should check for equality by matching references.

Reply:
good call - will make the change

Reply:
bump

Reply:
made the change and I agree with the change. in answer to your question - the intention behind that is "System.identityHashCode(...)" can't be overriden in the way ".equals()" and ".hashCode()" can be. 

That being said - I agree the likelihood of us running into trouble because someone overrode ".equals()" in a weird manner is very very low.